rambox.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+ <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since you provide this file it should be named properly from the start.

Reply:
I tried to lean flatpaks based on of other flathub flatpaks, mainly vscode and all files i found had names like com.visualstudio.code.metainfo.xml

I don't understand what I did wrong...

Reply:
the file should be named `app.rambox.Rambox.metainfo.xml`.
This include removing the `rename-appdata-file` and properly installing it.

You only use the rename if the file comes from upstream. Since it doesn't seem to then just make it right.

Reply:
or was @yselkowitz wrong

Reply:
named it back to app.rambox.Rambox based of feedback in the matrix chat

Reply:
gasinvein in matrix said:
> Older metainfo adopters used to have IDs in appname.desktop form, and some continue to use it for backward compatibility, but newely-created metainfos shouldn't do this.

Reply:
So why is the file still here?